packet/flatcar-linux/kubernetes/outputs.tf

```diff
@@ -5,3 +5,34 @@ output "kubeconfig-admin" {
 output "kubeconfig" {
   value = module.bootkube.kubeconfig-kubelet
 }
+
+output "dns_entries" {
+  value = concat(
+    # etcd
+    [
+      for device in packet_device.controllers:
+      {
+        name    = format("%s-etcd%d.%s.", var.cluster_name, index(packet_device.controllers, device), var.dns_zone),
```

Comment:
Can we please move `format` calls to either `locals` or to `null_resource`, so they are not duplicated?

Reply:
Oh yeah, I didn't want to use a null_resource but locals is a good solution :).